.mocharc.yaml

```diff
@@ -0,0 +1,12 @@
+full-trace: true
+check-leaks: true
+recursive: true
+timeout: 10000
+exit: true
+global:
```

Comment:
Can we add the comment here about being necessary for testing the NodeManager class?